pkg/composer/serviceparser/serviceparser.go

```diff
@@ -478,12 +478,6 @@ func newContainer(project *compose.Project, parsed *Service, i int) (*Container,
 	for _, v := range svc.DNS {
 		c.RunArgs = append(c.RunArgs, fmt.Sprintf("--dns=%s", v))
 	}
-
-	if len(svc.Entrypoint) > 1 {
-		return nil, fmt.Errorf("service %s: specifying entrypoint with multiple strings (%v) is not supported yet",
-			svc.Name, svc.Entrypoint)
-	}
```

Comment:
Could you also update https://github.com/containerd/nerdctl/blob/master/docs/compose.md#incompatibility ?

Reply:
Pushed a commit by myself https://github.com/containerd/nerdctl/pull/1070/commits/08924b9c54f4df48ce02090ad6c05f26c1bf4aa1